lib/communication/command_line.ex

```diff
@@ -1,8 +1,9 @@
-defmodule CommandLine do
+defmodule Communication.CommandLine do
```

Comment:
I think I see what you mean and agree with the more descriptive naming convention and better location, what do you think of something like `CommandLineIO` or `CommandLineInteractor` ?

Reply:
I will say I think Im leaning more `CommandLineCommunicator` as it avoids the use of `IO` and accurately describes the functionality (reading input and displaying output). It may seem a little redundant being under the communication directory, but I think its the best name so far

Reply:
Either of those are fine to me.

Reply:
Should be all updated following the new `CommandLineCommunicator` naming convention along with the new location